Tests/TuistCacheTests/Cache/CacheGraphMutatorTests.swift

```diff
@@ -592,6 +592,94 @@ final class CacheGraphMapperTests: TuistUnitTestCase {
         XCTAssertTrue(b.dependencies.contains(where: { $0.path == dFrameworkPath }))
         XCTAssertTrue(c.dependencies.contains(where: { $0.path == eXCFrameworkPath }))
     }
+    
+    // 10th scenario
+    //       +---->B (Framework)
```

Comment:
I think it may be worth adding a following scenario: `App +----> B (Framework) +-----> C (Bundle)`

Reply:
@fortmarek the test_map_when_eleventh_scenario is doing that.

Reply:
ahh, OK, I see. I think the ASCII graph is wrong then, though, as it implies that App depends on Bundle directly, and not transitively